### ISSUE TITLE
Add joblib optional dependency

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#903]
We have the following new deprecation warning from scikit-learn:

```
/Users/benjamin/.pyenv/versions/3.7.3/envs/yellowbrick3/lib/python3.7/site-packages/sklearn/externals/joblib/__init__.py:15
  /Users/benjamin/.pyenv/versions/3.7.3/envs/yellowbrick3/lib/python3.7/site-packages/sklearn/externals/joblib/__init__.py:15: 
  DeprecationWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 
  0.23. Please import this functionality directly from joblib, which can be installed with: 
  pip install joblib. If this warning is raised when loading pickled models, you may need to 
  re-serialize those models with scikit-learn 0.21+.
    warnings.warn(msg, category=DeprecationWarning)
```

Yellowbrick uses joblib to parallelize a few operations and had been relying on scikit-learn to maintain this dependency. If this is being removed from scikit-learn we should find out why and determine whether or not we should add it as an optional dependency in the same way scikit-learn is.

**Comment by rebeccabilbro** [comment]
I believe we can safely close this issue for now; we don't explicitly import `joblib` in any of our source files, so potentially it was a general-announcement-type warning from sklearn, which has since ceased.

**Comment by bbengfort** [comment]
Sorry about that, I'm not sure why I thought we were using joblib, maybe we removed it? It seems that ValidationCurve and LearningCurve have `n_jobs` parameters but those are being passed to scikit-learn. 

I had intended to use joblib with `KElbow` and `ManualAlphaSelection` to speed things up (and possibly also `RFECV`), maybe that's what I was thinking of? Anyway, glad this deprecation warning is gone!
